charts/fybrik/templates/openmetadata-connector/openmetadata-editor-role.yaml

```diff
@@ -0,0 +1,19 @@
+{{- $autoFlag := and .Values.coordinator.enabled (eq .Values.coordinator.catalog "openmetadata") }}
+{{- if include "fybrik.isEnabled" (tuple .Values.openmetadataConnector.enabled $autoFlag) }}
+# openmetadata-editor allows managing assets
+apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.clusterScoped }}
+kind: ClusterRole
+metadata:
+  name: {{ template "fybrik.fullname" . }}-openmetadata-editor-cr
+{{- else }}
+kind: Role
+metadata:
+  name: {{ template "fybrik.fullname" . }}-openmetadata-editor-role
+  namespace: {{ .Values.applicationNamespace | default .Release.Namespace  }}
+{{- end }}
+rules:
+- apiGroups: ["openmetadata.fybrik.io"]
+  resources: ["assets"]
```

Comment:
does openmetadata needs to access to assets?

Reply:
Currently, no. It only accesses assets stored in OpenMetadata.
In the future, it might.

Reply:
I would suggest removing it for now.

Reply:
removed. Thanks.